### PR TITLE
Debezium 0.10.0.CR1 release announcement

### PR DESCRIPTION
**Pull request by gunnarmorling**
@jpechane, here's the announcement. Can you merge it once the release is out?

### CODE REVIEW

File: blog/2019-09-10-debezium-0-10-0-cr1-released.adoc [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+i.e. you can now obtain the specific documentation applying for a particular release.
+
+The blog section of the website has been reworked, too:
+the main page shows an introductory snippet for the most recent posts,
+whereas a number of "featured" blog posts is listed on the left.
+These are typically earlier blog posts which explore advanced topics such as the link:/blog/2019/02/19/reliable-microservices-data-exchange-with-the-outbox-pattern/[outbox pattern] in depth and which we wanted to make easier to discover and consume.
+We hope you like the new website and documentation structure and it helps you to find all the information you're looking for better than before.
+If you run into any issues (formatting glitches, broken links etc.), please let us know.
+
+Refer to the link:/docs/releases/#release-0-10-0-cr1[0.10.0.CR1 release notes] for the complete list of addressed issues and the upgrading procedure.
```
Comment (gunnarmorling) [inline blog/2019-09-10-debezium-0-10-0-cr1-released.adoc]: @Naros, I'm unsure which link I should add here; could you help out?

Comment (Naros) [reply blog/2019-09-10-debezium-0-10-0-cr1-released.adoc]: `/releases/0.10/release-notes##release-0-10-0-cr1`

Comment (gunnarmorling) [reply blog/2019-09-10-debezium-0-10-0-cr1-released.adoc]: Thx, adjusted.

**Comment by gunnarmorling** [comment]
Closing, superseded by https://github.com/debezium/debezium.github.io/pull/353.

**Pull request closed without merging**